frontend/src/modules/dashboard/store/actions.js

```diff
@@ -5,6 +5,13 @@ import { ConversationService } from '@/modules/conversation/conversation-service
 import moment from 'moment'
 
 export default {
+  async reset({ dispatch }) {
+    dispatch('setFilter', {
```

Comment:
I think it should be `setFilters` instead of `setFilter` right?
At least according to the available action on the same file. I'm also getting a console error for the unknown action. Could you please confirm if it's ok? 🙏